src/containers/SplashPage/SplashPage.js

```diff
@@ -59,16 +59,20 @@ class SplashPage extends Component {
   render() {
     const { config } = this.props;
 
+    const integrationName = getConfigValue(config, 'integrationName');
+    const integrationNameCapital =
+      integrationName.charAt(0).toUpperCase() + integrationName.slice(1);
```

Comment:
It's a small detail but this will produce `Wordpress` rather than `WordPress`. Check if Garrett is OK with it.